src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities.cs

```diff
@@ -46,6 +55,78 @@ public static KeyValueLogScope CreateArtifactScope(string artifactType, IEndpoin
             return scope;
         }
 
+        public static AggregateSourceConfiguration GetTraceConfiguration(Models.TraceProfile profile, int metricsIntervalSeconds)
```

Comment:
https://github.com/dotnet/dotnet-monitor/pull/883#discussion_r713195397

Reply:
Yep, I meant to comment on that PR and apparently didn't; my plan is after I get all the actions in (so this PR and then the following one for Logs) to do some refactoring to improve organization and potentially improve the code reuse between the tests.